lib/polygeist/Passes/AffineCFG.cpp

```diff
@@ -332,7 +332,7 @@ AffineApplyNormalizer::AffineApplyNormalizer(AffineMap map,
              isValidSymbolInt(t.getDefiningOp()->getOperand(1))) &&
             (!(fix(t.getDefiningOp()->getOperand(0), false) &&
                fix(t.getDefiningOp()->getOperand(1), false)))) ||
-           (t.getDefiningOp<DivSIOp>() &&
+           ((t.getDefiningOp<DivUIOp>() || t.getDefiningOp<DivSIOp>()) &&
```

Comment:
If doing a divui I wonder if we should also check that the divisor is positive. The AffineExpr constraint assumes signed division iirc, which is why we originally didn't include.

Of course crashing is far worse, so we don't want that to occur.